app/__init__.py

```diff
@@ -379,17 +378,9 @@ def checkout(dbapi_connection, connection_record, connection_proxy):
                     'host': request.host,
                     'url_rule': request.url_rule.rule if request.url_rule else 'No endpoint'
                 }
-            # celery apps
-            elif current_task:
-                print(current_task)
-                connection_record.info['request_data'] = {
-                    'method': 'celery',
-                    'host': current_app.config['NOTIFY_APP_NAME'],  # worker name
-                    'url_rule': current_task.name,  # task name
-                }
```

Comment:
this needs to stay, as the `checkin` will still happen in celery tasks, and will crash if connection_record.info['request_data'] isn't set. the print can go though